src/SixLabors.Fonts/Tables/AdvancedTypographic/Gsub/LookupType5SubTable.cs

```diff
@@ -0,0 +1,359 @@
+        {
+            // SequenceContextFormat1
+            // +----------+------------------------------------+---------------------------------------------------------------+
+            // | Type     | Name                               | Description                                                   |
+            // +==========+====================================+===============================================================+
+            // | uint16   | format                             | Format identifier: format = 1                                 |
+            // +----------+------------------------------------+---------------------------------------------------------------+
+            // | Offset16 | coverageOffset                     | Offset to Coverage table, from beginning of                   |
+            // |          |                                    | SequenceContextFormat1 table                                  |
+            // +----------+------------------------------------+---------------------------------------------------------------+
+            // | uint16   | seqRuleSetCount                    | Number of SequenceRuleSet tables                              |
+            // +----------+------------------------------------+---------------------------------------------------------------+
+            // | Offset16 | seqRuleSetOffsets[seqRuleSetCount] | Array of offsets to SequenceRuleSet tables, from beginning of |
+            // |          |                                    | SequenceContextFormat1 table (offsets may be NULL)            |
+            // +----------+------------------------------------+---------------------------------------------------------------+
+            ushort coverageOffset = reader.ReadOffset16();
+            ushort seqRuleSetCount = reader.ReadUInt16();
+            ushort[] seqRuleSetOffsets = reader.ReadUInt16Array(seqRuleSetCount);
+            var seqRuleSets = new SequenceRuleSetTable[seqRuleSetCount];
+
+            for (int i = 0; i < seqRuleSets.Length; i++)
+            {
+                seqRuleSets[i] = SequenceRuleSetTable.Load(reader, offset + seqRuleSetOffsets[i]);
+            }
+
+            var coverageTable = CoverageTable.Load(reader, offset + coverageOffset);
+            return new LookupType5Format1SubTable(seqRuleSets, coverageTable);
+        }
+
+        public override bool TrySubstition(GSubTable gSubTable, IGlyphSubstitutionCollection collection, ushort index, int count)
+        {
+            int glyphId = collection[index][0];
+            if (glyphId < 0)
+            {
+                return false;
+            }
+
+            if (this.coverageTable.CoverageIndexOf((ushort)glyphId) > -1)
+            {
+                // TODO: Implement
+                // https://docs.microsoft.com/en-us/typography/opentype/spec/gsub#example-7-contextual-substitution-format-1
+                return false;
```

Comment:
@brianpopow I could do with your considerable spec parsing skills here if you have time. I can follow the spec to determine how to decode the tables however I can't make sense of how to actually implement the substitution. I've been trying t0 understand HarfBuzz but that's hieroglyphs to me.

Reply:
sure, cant promise I will understand it better, but I will take a look.

Reply:
there is alot to chew on here. It will take some time for me to understand that. I am currently trying to understand how harfbuzz does it, hope that will help here.